services/ui/ws/window_tree.cc

```diff
@@ -129,6 +129,7 @@ void WindowTree::DoOnEmbed(mojom::WindowTreePtr tree,
 
     ClientWindowId window_id;
     IsWindowKnown(root_window, &window_id);
+    AddActivationParent(window_id);
```

Comment:
Ok, I am fine with it for now, thank you maksim.

I would like to have a comment. Some thing like:

// In external window mode, every platform window is an activation parent.